sklearn/feature_selection/_rfe.py

```diff
@@ -142,6 +146,8 @@ def __init__(self, estimator, *, n_features_to_select=None, step=1,
                  verbose=0, importance_getter='auto'):
         self.estimator = estimator
         self.n_features_to_select = n_features_to_select
+        if n_features_to_select is not None and n_features_to_select < 0:
+            raise ValueError("n_features_to_select must be positive.")
```

Comment:
We prefer to validate during `fit`.

Reply:
I moved it. Did I understand correctly what you meant?

Reply:
Yup that works.

This line can be removed now.